test/team_test.rb

```diff
@@ -0,0 +1,75 @@
+require './lib/player'
+require './lib/team'
+require 'minitest/autorun'
+require 'minitest/pride'
+
+class TeamTest < MiniTest::Test
+
+  def test_team_exists
+    france = Team.new("France")
+
+    assert_instance_of Team, france
+  end
+
+  def test_it_has_a_country
+    france = Team.new("France")
+    expected = "France"
+    actual = france.country
+
+    assert_equal expected, actual
+  end
+
+  def test_france_starts_with_empty_team
+    france = Team.new("France")
+    expected = []
+    actual = france.players
+    assert_equal expected, actual
+  end
+
+  def test_france_starts_not_eliminated
+    france = Team.new("France")
+    expected = false
+    actual = france.eliminated?
+    assert_equal expected, actual
+  end
+
+  def test_france_becomes_eliminated
+    france = Team.new("France")
+    expected = true
+    actual = france.eliminated
```

Comment:
From the spec, we should be able to call `france.eliminated = false`. You can do this be defining the method `def eliminate=(new_value)`, or by using an attr_writer